tests/test_file_download.py

```diff
@@ -638,6 +638,28 @@ def test_download_from_a_gated_repo_with_hf_hub_download(self):
                     cache_dir=tmpdir,
                 )
 
+    def test_keep_lock_file(self):
+        """Lock files should be kept if HF_HUB_KEEP_LOCK_FILES is True
+        """
+        with SoftTemporaryDirectory() as tmpdir:
+            with patch("huggingface_hub.file_download.HF_HUB_KEEP_LOCK_FILES", False):
+                hf_hub_download(DUMMY_MODEL_ID, filename=CONFIG_NAME, cache_dir=tmpdir)
+            for subdir, dirs, files in os.walk(tmpdir):
+                for file in files:
+                    # No lock files should exist
+                    self.assertNotRegex(file, ".*\.lock")
+
+        with SoftTemporaryDirectory() as tmpdir:
+            with patch("huggingface_hub.file_download.HF_HUB_KEEP_LOCK_FILES", True):
+                    hf_hub_download(DUMMY_MODEL_ID, filename=CONFIG_NAME, cache_dir=tmpdir)
```

Comment:
```suggestion
                hf_hub_download(DUMMY_MODEL_ID, filename=CONFIG_NAME, cache_dir=tmpdir)
```

(formatting issue)

Reply:
yeah, found `black` command in github actions, doing a format now.